crates/diesel_models/src/refund.rs

```diff
@@ -205,3 +205,10 @@ pub struct RefundCoreWorkflow {
     pub merchant_id: String,
     pub payment_id: String,
 }
+
+#[derive(Debug, Eq, PartialEq, Deserialize, Serialize)]
+pub struct AutoRefundWorkflow {
+    pub payment_intent: crate::payment_intent::PaymentIntent,
```

Comment:
Can we use payment attempt over here since it has the `connector_transaction_id` .

Reply:
Yup. Okay.